### DISCUSSION TITLE
Node synchronization completed, transfer no transaction information

### DISCUSSION THREAD

**Discussion by wbm0701** [Chia-Network/chia-blockchain#5318] [Support]
Why does my node complete synchronization and transfer to the generated address? After a long time, I can't find the balance and transaction record. I guarantee that it is the main network and the synchronization is complete

Current Blockchain Status: Full Node Synced

Peak: Hash: bcc2645ea35d66ba8a950f4ad00aaa87b07aa4e85313138921a8ebe5cce50497
      Time: Mon May 17 2021 10:59:44 CST                  Height:     292782

Estimated network space: 5.839 EiB
Current difficulty: 394
Current VDF sub_slot_iters: 113246208
Total iterations since the start of the blockchain: 945728861181

  Height: |   Hash:
   292782 | bcc2645ea35d66ba8a950f4ad00aaa87b07aa4e85313138921a8ebe5cce50497
   292781 | cbba3d93c3e8e8405c2f22c1bb1c2a0751f1e20d24cd4eab83101f20375c6c48
   292780 | 278feb661a93386d28d1fc33aea5ae9a6c2557917ab743ddb648ae334057303c
   292779 | c6ff6d14a2ced7f3d5e547596e88567754a8cce6ce2d2f14567fef39edc041e8
   292778 | 9b99140a3f6fbdc28ca67f1756141de4542b21876cf7484be6669c2e2b7fdc59
   292777 | 2a70c20bbc74db1583335f0c8f4e76fae48650814adebf7fd7796efc0cfbb808
   292776 | 91f2ca40e958afce826f40b46738f14c0e122842e982e5c122293c76b61bc4bd
   292775 | db6e531ea9a8dea74fcd8d4329145786b23626752c98ed4f612f01fe37bfd15a
   292774 | 2d50076ba6967a01b0c6b14516409c2944fcb9df4e730710af3a5956353ed97d
   292773 | ffc04fc0923ffc3f38f0c6abbfb24719aca01b982e10238ea96d3d6047b1375b

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
